src/styles/index.css

```diff
@@ -23,3 +23,7 @@ svg.recharts-surface {
 .recharts-cartesian-grid-horizontal line:last-child {
   stroke-opacity: 0;
 }
+
+.foreignNote strong {
+  font-weight: 700;
+}
```

Comment:
This should not be needed. @buberdds do you remember why we have https://github.com/oasisprotocol/explorer/blob/c46707429ec456c4d4badf9c7c2421eaa04cf327/src/styles/theme/defaultTheme.ts#L110

Reply:
I'm happy to simplify, if we find a good way.

Reply:
I would change it here. I only noticed it affected "Type     Transfer" in  https://explorer.dev.oasis.io/mainnet/emerald/transactions/0x2d3f414f402eb83e99a59ae7b08d849e05884f36bacf5e2d2cc8b967566176cf

Reply:
OK